templates/upload.html

```diff
@@ -42,7 +42,7 @@ <h2>Instructions</h2>
                 <li><a href="https://en.wikipedia.org/wiki/Thermal_comfort#Cooling_Effect">Cooling Effect (CE)</a></li>
             </ul>
             A file named "results.csv" will be automatically downloaded by your browser which contains all the above indexes.
-            </p>
+{#            </p>#}
```

Comment:
Is this intentional?

Reply:
yes, Internet Explorer was telling me that </p> was not needed